web/modules/custom/joinup_core/joinup_core.module

```diff
@@ -490,6 +490,27 @@ function joinup_core_entity_create(EntityInterface $entity) {
   }
 }
 
+/**
+ * Implements hook_entity_insert().
+ */
+function joinup_core_entity_insert(EntityInterface $entity) {
```

Comment:
We should not mix UI with API. The message should be triggered only in UI context, but `hook_entity_insert()` is a general API hook. This means the message will be triggered even a module is adding a collection/solution via code or when we import/federate solutions. We don't want this message to be displayed in such conditions. That's why we should hook in the form API flow rather than in the entity API. Probably, you'll need to add a new submit callback in `joinup_core_form_rdf_entity_form_alter()`, in certain conditions (i.e. check the bundle) and popup the message in that callback.

Reply:
Alright! I've implemented it.